vita3k/util/include/util/log.h

```diff
@@ -63,24 +57,33 @@ ExitCode add_sink(const fs::path &log_path);
     LOG_CRITICAL(__VA_ARGS__)
 
 #define LOG_ONCE(log_function, ...)    \
-    do {                               \
+    {                                  \
         static bool LOG_DONE = false;  \
         if (!LOG_DONE)                 \
             log_function(__VA_ARGS__); \
         LOG_DONE = true;               \
-    } while (0)
+    }
```

Comment:
then `;` after block become a warning. Such loop is a special kind of expression to use in macros, it is a block after that `;` should be (after `{ ... }` should not be a `;`)

Reply:
Oh, I didn't know ty, I will change that the the way it was previously.